pages/home/glossary.mdx

```diff
@@ -87,9 +87,9 @@ A 'user' is your end-user, the person who is using your application or agent.  U
 
 Monthly Active Users are the unique end-users (counted by `user_id`) who have executed a tool in your app/agent within the past month.  If the same `user_id` calls a tool multiple times in the same month, or executes multiple tools, it is only counted once.
 
-### Authentication Starts (Auth Starts)
+### User Authentications (User Auths)
```

Comment:
I thought we decided to call these "token introspections" I think?  
CC @nbarbettini